caf: ble_adv: Add known issue for NCSDK-18587 (#9660)
nrfconnect/sdk-nrf: doc/nrf/known_issues.rst

  **Workaround:** Manually cherry-pick and apply commit with fix from main (commit hash: ``c61c677872943bcf7905ddeec8b24b07ae50752e``).

.. rst-class:: v2-2-0

NCSDK-18587: :ref:`caf_ble_adv` leaves off state on peer disconnection

>   If Bluetooth peer disconnects while the module is in off state, the Bluetooth LE advertising module enters ready state.
>   The module shall not leave the off state until :c:struct:`wake_up_event` is received.

Reviewer: ```suggestion
  If Bluetooth peer disconnects while the module is in the off state, the Bluetooth LE advertising module enters the ready state.
  The module must remain in the off state until :c:struct:`wake_up_event` is received.
```
As the two known issues mentioned are the same and the issue is related to CAF libraries, we could have this in one place, under the "Common Application Framework (CAF)" section.
